spec/services/preserved_object_handler_check_exist_spec.rb

```diff
@@ -162,12 +162,6 @@
                 po_handler.check_existence
                 expect(pc.reload.updated_at).to be > orig
               end
-              it 'status becomes "ok" if it was invalid_moab (b/c after validation)' do
-                pc.status = PreservedCopy::INVALID_MOAB_STATUS
-                pc.save!
-                po_handler.check_existence
-                expect(pc.reload.status).to eq PreservedCopy::OK_STATUS
-              end
```

Comment:
I think if we are performing a validation anyway, and it is staring from INVALID_MOAB, we should be able to set it to "ok"

Reply:
i think that the way it's implemented in this PR, things are halting before moab validation is performed.  but i'd be totally open to something slightly more clever, where a status of `invalid_moab` triggered a moab validation, and marked things `ok` and proceeded if no errors were detected.

on the other hand, it did seem like the idea was to punt on any sort of clever remediation detection, which does make me slightly hesitant to add that bit of nuance.  similarly, if the only problem were an unexpected version error, we could see if the version on disk equal to or greater than the version expected by the catalog.  but my recollection is that punting on that was the genesis of this ticket.  and they feel like the same general category of error to me, i think (i.e., errors that need manual attention, where we don't expect our current processes to repair the automatically).  i guess some types of moab validation failure are not things that you'd want to block further checking... maybe there should be a threshold of moab validation error?  maybe that's getting too clever.

i'm not sure i'm convinced that "validate and set to ok if it is" the right way to go, but i'm also definitely not sure this PR's approach is the right way to go.  i'm also worried there will be unforseen pitfalls in the auto-mark-ok approach that will come out once we start testing and using the code, which is one of the things that made "skip and manually remediate status" so attractive to me in the first place.

maybe face to face discussion after break?  seems unlikely this will get merged this afternoon anyway?